feature/mission_recommendation/src/main/java/com/lgtm/android/mission_recommendation/ui/SuggestionDashboardViewModel.kt

```diff
@@ -0,0 +1,52 @@
+package com.lgtm.android.mission_recommendation.ui
+
+import android.content.ContentValues.TAG
+import android.util.Log
+import androidx.lifecycle.LiveData
+import androidx.lifecycle.MediatorLiveData
+import androidx.lifecycle.MutableLiveData
+import androidx.lifecycle.viewModelScope
+import com.google.firebase.crashlytics.ktx.crashlytics
+import com.google.firebase.ktx.Firebase
+import com.lgtm.android.common_ui.base.BaseViewModel
+import com.lgtm.domain.constants.Role
+import com.lgtm.domain.mission_suggestion.SuggestionContent
+import com.lgtm.domain.repository.AuthRepository
+import com.lgtm.domain.usecase.SuggestionUseCase
+import dagger.hilt.android.lifecycle.HiltViewModel
+import kotlinx.coroutines.launch
+import javax.inject.Inject
+
+@HiltViewModel
+class SuggestionDashboardViewModel @Inject constructor(
+    private val suggestionUseCase: SuggestionUseCase,
+    authRepository: AuthRepository
+): BaseViewModel() {
+
+    val suggestionBtnVisibility: Boolean = authRepository.getMemberType() == Role.REVIEWEE
+
+    private val _suggestionList = MutableLiveData<List<SuggestionContent>>()
+    val suggestionList: LiveData<List<SuggestionContent>> = _suggestionList
+
+    private val _dashBoardEmptyVisibility = MediatorLiveData<Boolean>()
+    init {
+        _dashBoardEmptyVisibility.addSource(_suggestionList) {
+            _dashBoardEmptyVisibility.postValue(it.size <= 1)
+        }
+    }
+    val dashBoardEmptyVisibility: LiveData<Boolean> = _dashBoardEmptyVisibility
```

Comment:
`MediatorLiveData`는 원래 두개 이상의 liveData를 같이 observe하고 싶을때 사용하는걸루 알고있는데
혹시 이거 쓴 내가 모르는 이유가 있을까!?

```suggestion

    private val isDashBoardEmpty = (suggestionList.value?.size ?: 0) <= 1
    
    private val _dashBoardEmptyVisibility = MediatorLiveData(isDashBoardEmpty)
    val dashBoardEmptyVisibility: LiveData<Boolean> = _dashBoardEmptyVisibility
```

이렇게 수정하면 꼭 `MediatorLiveData` 안쓰고도 할 수 있을것같아!